### DISCUSSION TITLE
重启 简化首页 功能，屏蔽轮播图

### DISCUSSION THREAD

**Discussion by MJSummer** [the1812/Bilibili-Evolved#5038] [功能建议 / Ideas]
[v2.9.5](https://github.com/the1812/Bilibili-Evolved/discussions/5024)中废弃了 简化首页 功能，推荐使用 [BewlyBewly](https://github.com/BewlyBewly/BewlyBewly) 替代。
然而BB对首页UI进行了较大程度的改动，部分功能与BE存在冲突，
元素空白过多导致阅览效率下降，且不支持2k分辨率175%缩放的5列展示。

因此，希望能够重启 简化首页 功能，能够屏蔽首页轮播图即可。

**Comment by the1812** [comment]
轮播图可以考虑提取独立的功能，简化首页废弃是因为其初衷是自定义首页展示的各个分区，现在新版首页变成瀑布流了这个功能就没有用了

**Reply by Geeyun-JY3** [reply]
借机捞一下 #4075，思路类似（
